### PR TITLE
Use new, simpler codelists API

### PR DESCRIPTION
**Pull request by evansd**

### CODE REVIEW

File: analysis/codelists.py [hunk 1]
```diff
@@ -1,102 +1,71 @@
...
+long_covid_combine = (
+    long_covid_nice_dx
+    + long_covid_referral_codes
+    + long_covid_assessment_codes
 )
 
 # https://www.opencodelists.org/codelist/opensafely/covid-identification-in-primary-care-probable-covid-sequelae/2020-07-16/
 covid_primary_care_sequelae = codelist_from_csv(
     "codelists/opensafely-covid-identification-in-primary-care-probable-covid-sequelae.csv",
-    system="ctv3",
```
Comment (hendersonad) [inline analysis/codelists.py:52]: I am getting the following error message when trying to run the code. Is there something I need to do to make this new API work or should I add the system argument back in? 

`     Traceback (most recent call last):
       File "/workspace/analysis/dataset_definition_cases.py", line 5, in <module>
         import codelists
       File "/workspace/analysis/codelists.py", line 6, in <module>
         long_covid_nice_dx = codelist_from_csv(
     TypeError: codelist_from_csv() missing 1 required positional argument: 'system'`

**Review by hendersonad — COMMENTED** [review]
Thank you for the edits, appreciate it. I am hitting an error in the `codelist_from_csv()` function now (comment [here](https://github.com/opensafely/openprompt-vaccine-long-covid/pull/3/files#r1120432587) )

Any advice is very welcome, thank you

**Comment by evansd** [comment]
Oh, sorry Alasdair! This is very confusing on my part. We're making some simplifications to the way codelists work but that version of Data Builder hasn't actually been released yet. I was pre-emptively making PRs for all ongoing projects so I could test that the new changes all worked.

These are marked as "Draft" in Github so you can't actually merge them, but I'd forgotten that you'd still get notifications for them.

Once we release the new version then the tests here should start passing and I'll mark the PR as ready to merge and let you know.

Until then, feel free to ignore it and sorry again for the confusion.

**Comment by evansd** [comment]
@hendersonad This is ready to merge now. Sorry for the earlier confusion!

**Pull request merged**